app/main.py

```diff
@@ -52,14 +57,22 @@ def __init__(self):
                 self._rabbit_hosts[1],
                 self._rabbit_port) + settings.HEARTBEAT_INTERVAL,
         ]
+        
+        self._aws_url = [
```

Comment:
We should rename these in line with the change to naming the queues i.e. `_eq_queue_urls` and `_sdx_queue_url`. `_sdx_queue_url` should just be a string, if we need to pass an array of strings when instantiating `QueuePublisher` then you can do `urls=[self._aws_url]`.